client/webserver/webserver.go

```diff
@@ -57,22 +57,22 @@ var (
 type clientCore interface {
 	Exchanges() map[string]*core.Exchange
 	Register(*core.RegisterForm) error
-	Login(pw string) ([]*db.Notification, error)
-	InitializeClient(pw string) error
+	Login(pw []byte) ([]*db.Notification, error)
+	InitializeClient(pw []byte) error
```

Comment:
As per other comment, I prefer them to stay like this, but **dexc** won't build right now because these don't match **Core**. 

In general, you should build **dexc** and test live on simnet any time changes are made to **webserver** or **core**.

Reply:
Yeah... wonder how I missed that...